### ISSUE TITLE
Possible cause for soundhax error

### ISSUE THREAD

**Issue by GarretXing** [nedwill/soundhax#72]
When ever I extract the files from the homebrew starter kit themes is not in it, and in all the videos of using the hack have the themes folder.  I can't seem to find out how to get the themes folder, it never shows up.

**Comment by GarretXing** [comment]
I think it may be the fault of the computer no the 3ds

**Comment by Jeanolos** [comment]
No. The Themes folder isn't necessary. Just create it yourself. Important are the 3ds folder and the boot.3dsx

**Comment by GarretXing** [comment]
Ok thanks

**Comment by GarretXing** [comment]
I also tried with my old 3ds and it did'nt work on that one either
